### DISCUSSION TITLE
后续是否支持下MP4导出，方便分享到外部~

### DISCUSSION THREAD

**Discussion by haywaa** [Tencent/libpag#427] [Ideas] [answered]
从AE文件转PAG后，后续可能在服务端进行图片文本替换等二次编辑。后续是否可以支持下导出PNG，这样就可以将视频进行分享了（而不是在内部应用进行查看）

**Comment by kevingpqi123** [comment, marked as answer]
从目前来看 PAG 层面不会做这件事， PAG 并不是视频编辑框，它只负责动效的渲染的部分，后续也只会专注于这一领域。视频编辑框架是另一个范畴的技术领域，需要业务方自己开发或者接入行业里成熟的方案，解决视频编解码，合成，导出等能力。
